entrypoint.py

```diff
@@ -24,23 +26,128 @@ def age(self, birthdate):
 
         return age
 
-    def check_file_content(self, url, columns):
-        """If the name of the columns in the uploaded csv file are not consisted with
-        the name of variables in metadata manager, file shouldn't be processed.
-        """
-        import requests
-        import json
+    import pandas as pd
```

Comment:
It looks like that this import is not used, if that is the case please remove it